dream_demo/bin/dune

```diff
@@ -8,4 +8,14 @@
 (rule
   (targets main.ml)
   (deps main.eml.ml)
+  (action (run dream_eml %{deps} --workspace %{workspace_root})))
```

Comment:
Sorry about the auto-formatting that was applied to this file. Nothing was changed in this file though.